cmake/modules/hunter_autotools_project.cmake

```diff
@@ -245,9 +245,9 @@ function(hunter_autotools_project target_name)
       # Extra space at the end of the arch_flags is needed below when appending
       # to configure_opts, please do not remove!
       if(is_simulator)
-        set(arch_flags "-arch ${ios_architecture} -isysroot ${IPHONESIMULATOR_SDK_ROOT} -miphoneos-version-min=${IOS_SDK_VERSION} ")
+        set(arch_flags "-arch ${ios_architecture} -isysroot ${IPHONESIMULATOR_SDK_ROOT} -miphoneos-version-min=${CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET} ")
```

Comment:
I think it would be nice to add check that this variable is not empty:
```cmake
if("${CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET}" STREQUAL "")
  hunter_user_error("CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET is not set for iOS")
endif()
```

What package have you used for testing? You have to update `PACKAGE_INTERNAL_DEPS_ID` of this package.

Reply:
> I think it would be nice to add check that this variable is not empty:

Yep, full ack. I fixed these. See my new commit.

> What package have you used for testing? You have to update PACKAGE_INTERNAL_DEPS_ID of this package.

Hmm, I'm not sure if it was openssl build..... let me figure out.

Reply:
> I'm not sure if it was openssl build..... let me figure out

OpenSSL is not using this code